test/e2e/shared/aws.go

```diff
@@ -798,24 +798,27 @@ type ServiceQuota struct {
 	RequestStatus       string
 }
 
-func EnsureServiceQuotas(sess client.ConfigProvider) map[string]*ServiceQuota {
+func EnsureServiceQuotas(sess client.ConfigProvider) (map[string]*ServiceQuota, map[string]*servicequotas.ServiceQuota) {
 	limitedResources := getLimitedResources()
 	serviceQuotasClient := servicequotas.New(sess)
 
+	originalQuotas := map[string]*servicequotas.ServiceQuota{}
+
 	for k, v := range limitedResources {
 		out, err := serviceQuotasClient.GetServiceQuota(&servicequotas.GetServiceQuotaInput{
 			QuotaCode:   aws.String(v.QuotaCode),
 			ServiceCode: aws.String(v.ServiceCode),
 		})
 		Expect(err).NotTo(HaveOccurred())
+		originalQuotas[k] = out.Quota
```

Comment:
How about we only save `value` field from below:

```
classiclb:
  Adjustable: true
  ErrorReason: null
  GlobalQuota: false
  Period: null
  QuotaArn: arn:aws:servicequotas:us-west-2:705628986737:elasticloadbalancing/L-E9E9831D
  QuotaCode: L-E9E9831D
  QuotaName: Classic Load Balancers per Region
  ServiceCode: elasticloadbalancing
  ServiceName: Elastic Load Balancing (ELB)
  Unit: None
  UsageMetric:
    MetricDimensions:
      Class: None
      Resource: ClassicLoadBalancersPerRegion
      Service: Elastic Load Balancing
      Type: Resource
    MetricName: ResourceCount
    MetricNamespace: AWS/Usage
    MetricStatisticRecommendation: Maximum
  Value: 20
```

Reply:
I did think about that but thought there was no harm in saving everything...just in case.